homepage/models.py

```diff
@@ -69,6 +69,15 @@ class Req_Code(models.IntegerChoices):
     def __str__(self):
         return f'req_cour = {self.req_course_id}, desired_cour = {self.course_id}, req_code = {self.req_code}'
 
+    @staticmethod
+    def get_prerequisites_for_course(course):
+        preqs = Prerequisites.objects.filter(course_id=course)
+        return preqs
+
+    @staticmethod
+    def does_course_have_prerequisites(course):
+        return Prerequisites.objects.filter(course_id=course).exists()

```

Comment:
maybe you can use the previous function instead of repeat the same code
```suggestion
        return get_prerequisites_for_course(course).exists()
```